com.spotify.Client.json

```diff
@@ -17,12 +19,17 @@
         "--device=dri",
         "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
         "--talk-name=org.gnome.SessionManager",
+        "--talk-name=org.kde.StatusNotifierWatcher",
         "--own-name=org.mpris.MediaPlayer2.spotify",
         "--filesystem=xdg-music:ro",
         "--filesystem=xdg-pictures:ro",
         "--env=LD_LIBRARY_PATH=/app/lib",
         "--env=TMPDIR=/tmp"
     ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/vala/bin",
+        "prepend-ld-library-path": "/usr/lib/sdk/vala/lib"
+    },
```

Comment:
Why are you including Vala? Surely every dep supports disabling it?

Reply:
I ended up merging this with Vala removed.

Reply:
In my testing `ayatana-ido` failed to build without vala. If it works without it then it's great.

Reply:
I had to patch it :disappointed:

Reply:
I just added it to Liferea on GNOME 45 without vala and it seems to work fine? No patching needed. https://github.com/flathub/net.sourceforge.liferea/commit/071f44ed8cc8bb0813177559d86efac1141ac14d

Reply:
Ah nvm, forgot that one comes with vala